corehq/apps/es/aggregations.py

```diff
@@ -547,49 +547,53 @@ class NestedTermAggregationsHelper(object):
         terms=[
             AggregationTerm('entry_id', 'entry_id'),
         ],
-        bottom_level_aggregation=SumAggregation('balance', 'balance'),
+        inner_most_aggregation=SumAggregation('balance', 'balance'),
     ).get_data()
     """
 
-    def __init__(self, base_query, terms, bottom_level_aggregation=None):
+    def __init__(self, base_query, terms, inner_most_aggregation=None):
         self.base_query = base_query
         self.terms = terms
-        self.bottom_level_aggregation = bottom_level_aggregation
+        self.inner_most_aggregation = inner_most_aggregation
 
-    def get_data(self):
+    @property
+    def query(self):
         previous_term = None
+
+        if self.inner_most_aggregation is not None:
+            if not isinstance(self.inner_most_aggregation, SumAggregation):
+                raise ValueError('currently only SumAggregations and its children are supported')
```

Comment:
There are some aggregations like `MinAggregation` that inherit from `SumAggregation` now. They should still work in this helper although I haven't explictly tested that.